### PR TITLE
added read_cam.py

### PR DESCRIPTION
**Pull request by SegFault03**

### CODE REVIEW

File: src/models/read_cam.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import cv2
+import imutils
+import numpy as np
+
+# global variables
+bg = None
```
Comment (soumyadeepkun) [inline src/models/read_cam.py:6]: Use capital letters for global variables.

Ex: 
```python
BG = None
```

File: src/models/read_cam.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        # observe the keypress by the user
+        keypress = cv2.waitKey(1) & 0xFF
+
+        # if the user pressed "q", then stop looping
+        if keypress == ord("q"):
+            break
+
+# free up memory
+camera.release()
+cv2.destroyAllWindows()
```
Comment (soumyadeepkun) [inline src/models/read_cam.py:113]: keep these lines under indent of function for `"__main__"`

**Review by soumyadeepkun — CHANGES_REQUESTED** [review]
Incorporate the suggested changes, and use [type annotations](https://docs.python.org/3/library/typing.html) for specifying return-types of functions.

**Comment by soumyadeepkun** [comment]
Also, update `requirements.txt` to reflect list of required non-standard libraries and dependencies. 
Use `pipreqs` or any other appropriate package to update the list.

**Comment by SegFault03** [comment]
It will be difficult to specify type-annotations since I genuinely have no idea what is the type of objects that are being used by (some) functions. 

As for the other requested changes, I'll see what I can do.

**Pull request merged**